src/UglyToad.PdfPig/Graphics/Operations/PathConstruction/BeginNewSubpath.cs

```diff
@@ -37,6 +37,7 @@ public void Run(IOperationContext operationContext)
         {
             operationContext.BeginSubpath();
             operationContext.CurrentPosition = Point;
+            operationContext.CurrentPath.LineTo(Point.X, Point.Y);
```

Comment:
This doesn't quite seem like the right behavior here, the spec says:

> Begin a new subpath by moving the current point to coordinates (x, y), omitting any connecting line segment. If the previous path construction operator in the current path was also m, the new m overrides it; no vestige of the previous m operation remains in the path.

It might be we need to support some other operation on the `CurrentPath` object.

Reply:
I think it's fine. There's logic in LineTo that actually does MoveTo when current position is not set yet.